Backend/DataAccessLayer/Database/dbContext.cs

```diff
@@ -11,9 +11,16 @@ namespace DataAccessLayer.Database
 {
     public class DatabaseContext : DbContext
     {
+        // TODO: turn into enviornmental variables for dev and deploy
+        const string LOCAL_SQL_SERVER = "(localdb)\\MSSQLLocalDB";
+        const string LOCAL_DB_NAME = "NightWatchDB";

```

Comment:
Excellent - Please turn these into env variables before this is merged.

Reply:
maybe those can be added at the web deploy config